client/controller/shared.js

```diff
@@ -68,7 +68,7 @@ export function setSection( section ) {
 	return ( context, next = noop ) => {
 		context.store.dispatch( setSectionAction( section ) );
 
-		loadSectionCSS( context, next );
```

Comment:
We need that change so the CSS is loaded when we change section.

Reply:
> Otherwise we might load ltr and then rtl css

I agree but there is no clean solution currently, if we remove it we won't load the section's css when changing section.
I created a task on our board to create a `ui.locale` reducer so we can have the ui current language separated from the user info or the query param. This way, the server doing SSR will be able to set this value in initialReduxStore and pass it to the client.

Reply:
Maybe I didn't understand you correctly, but we might need to push that to prod before merging this patch.
I might've been not entirely clear on what I mean, so I'll try to elaborate on that a bit.

Consider the scenario user is visiting `/log-in/he`, the server figures out it needs to send `login.rtl.css` and it 
indeed sends it:
```html
...
<link rel="stylesheet" id="section-css" href="/calypso/sections-rtl/login.rtl.css?v=81d380f1dc">
</head>
...
```

which is as it should be.

Browser downloads the JS and executes it, calling `setSection()` and so on,
but ohhh no! 😭  we don't yet have `context.isRTL` set, because `setUpLocale` wasn't called,
so we hit:

![screen shot 2017-09-26 at 13 30 26](https://user-images.githubusercontent.com/326402/30855898-373cbdce-a2bf-11e7-9b10-913de82d71e2.png)

![screen shot 2017-09-26 at 13 33 18](https://user-images.githubusercontent.com/326402/30855952-70b82386-a2bf-11e7-970e-07fef251cf94.png)
which is the wrong file.

So we either delay css loading to `setUpLocale()` by calling `next()` here,
or we go to the previous method of having us insert a middleware to `page` at `bundler/loader`.

otherwise we end up having 2 CSSs loaded and actually the wrong one being used in the end as in the example here.

Sorry for the long comment.

How you think we should proceed @Tug ?